src/open_inwoner/pdc/managers.py

```diff
@@ -26,6 +26,10 @@ def draft(self):
 
     def visible_for_user(self, user: User):
         if user.is_authenticated:
+            if bool(getattr(user, "bsn", None)):
```

Comment:
Is the cast to `bool` necessary (why not just test for truthy)?

Reply:
You're right, not sure why I casted it, so I removed this